### PR TITLE
[Enhancement][RHEL/7] STIG updates for yum

### LINKED ISSUE #1122: Create RHEL7 RULE for localpkg_gpgcheck

### LINKED ISSUE #1123: Create RHEL7 rule for repo_gpgcheck

### LINKED ISSUE #1124: Create RHEL7 rule for clean_requirements_on_remove
image

### PR DESCRIPTION
**Pull request by redhatrises**
- Fixes #1122
- Fixes #1123
- Fixes #1124

### CODE REVIEW

File: RHEL/7/input/oval/clean_components_post_updating.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <criteria>
+      <criterion comment="check value of clean_requirements_on_remove in /etc/yum.conf" test_ref="test_yum_clean_components_post_updating" />
+    </criteria>
+  </definition>
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="check value of clean_requirements_on_remove in /etc/yum.conf" id="test_yum_clean_components_post_updating" version="1">
+    <ind:object object_ref="object_yum_clean_components_post_updating" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="object_yum_clean_components_post_updating" comment="clean_requirements_on_remove set in /etc/yum.conf" version="1">
+    <ind:filepath>/etc/yum.conf</ind:filepath>
+    <ind:pattern operation="pattern match">^\s*clean_requirements_on_remove\s*=\s*1\s*$</ind:pattern>
```
Comment (shawndwells) [inline RHEL/7/input/oval/clean_components_post_updating.xml]: Should accept `````=1`````, `````=True`````, `````=yes`````

Ref http://linux.die.net/man/5/yum.conf

**Comment by shawndwells** [comment]
Looks good! Two minor things:

- OVAL regex should accept =1|True|yes

- Remove newly assigned CCEs from the available list

**Comment by redhatrises** [comment]
Looks good! Two minor things:
@shawndwells 
>- OVAL regex should accept =1|True|yes

Done

>- Remove newly assigned CCEs from the available list

Apparently forgot to `git add` the file. Fixed.

**Comment by mpreisler** [comment]
ACK

**Pull request merged**